src/relayServerUtils.ts

```diff
@@ -359,6 +396,47 @@ async function validateExpirationTime(
   }
 }
 
+type VerifierQueries = 'Token' | 'Contract';
+
+async function callVerifierMethod(
```

Comment:
@franciscotobar  I find this a bit confusing, so I split the function into 2 separate ones.